src/index.ts

```diff
@@ -1,29 +1,50 @@
 import {Command, flags} from '@oclif/command'
+import { readFileSync, writeFileSync } from 'fs';
+
+
+
+const depExtractor = (file: any, depOrDevDep: string)=>{
+  for(let dep in file.jspm[depOrDevDep]){ // going over devDep or dep in jspm
+    const splitVrsnName = file.jspm[depOrDevDep][dep].split('@');
+    
+    const gitOrNpm = splitVrsnName[0].split(':')
+    const depName = gitOrNpm[0] == 'npm' ? gitOrNpm[1] : dep;
+
+    const depVersion = splitVrsnName[1]; 
+    file[depOrDevDep][depName] = depVersion; //inserting dependency 
```

Comment:
@noa-moshe This can crash if there is a devDependencies in the JSPM section but not in the main file, would be worth capturing these edge cases

Reply:
Basically making sure the section exist in the original file and if not create them